plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -3082,8 +3104,11 @@ public void testIllegalStorageFormatDuringTableScan()
         // to make sure it can still be retrieved instead of throwing exception.
         try (Transaction transaction = newTransaction()) {
             ConnectorMetadata metadata = transaction.getMetadata();
-            Map<SchemaTableName, List<ColumnMetadata>> allColumns = metadata.listTableColumns(newSession(), new SchemaTablePrefix(schemaTableName.getSchemaName()));
-            assertTrue(allColumns.containsKey(schemaTableName));
+            assertEquals(
+                    metadata.streamTableColumns(newSession(), new SchemaTablePrefix(schemaTableName.getSchemaName()))
+                            .filter(result -> result.getTable().equals(schemaTableName))
+                            .count(),
+                    1);
```

Comment:
Use `assertThat` so that, should this fail on CI one day, the exception message produced would be uiseful